src/plugins/intel_gpu/src/graph/strided_slice.cpp

```diff
@@ -201,8 +202,7 @@ void strided_slice_inst::update_output_memory() {
     if (static_cast<bool>(_outputs[0]) && _network.get_engine().is_the_same_buffer(output_memory(), input_memory()))
         return;
 
-    if (_node != nullptr)
-        build_deps();
+    build_deps();
```

Comment:
we have similar pattern in many places of source code on `graph/*`. Could you check the required behavior and fix other parts as well?

Reply:
updated code&comment